clients/admin-ui/src/features/common/privacy-notice-regions.ts

```diff
@@ -0,0 +1,120 @@
+import { PrivacyNoticeRegion } from "~/types/api";
+
+/**
+ * A mapping of privacy notice regions to human readable names.
+ *
+ * The typing is important here so that our build will fail if we are missing a region.
+ */
```

Comment:
to show with an example, if I were to add something to the `PrivacyNoticeRegion` enum but not update it here, we'd get this error on build (and therefore in CI)

```
./src/features/common/privacy-notice-regions.ts:8:14
Type error: Property '[PrivacyNoticeRegion.AK]' is missing in type '{ us: string; us_al: string; us_ak: string; us_az: string; us_ar: string; us_ca: string; us_co: string; us_ct: string; us_de: string; us_fl: string; us_ga: string; us_hi: string; us_id: string; us_il: string; ... 84 more ...; ca_yt: string; }' but required in type 'Record<PrivacyNoticeRegion, string>'.

   6 |  * The typing is important here so that our build will fail if we are missing a region.
   7 |  */
>  8 | export const PRIVACY_NOTICE_REGION_RECORD: Record<PrivacyNoticeRegion, string> =
     |              ^
   9 |   {
  10 |     [PrivacyNoticeRegion.US]: "United States",
  11 |     [PrivacyNoticeRegion.US_AL]: "Alabama (USA)",
npm ERR! Lifecycle script `build` failed with error: 
npm ERR! Error: command failed 
npm ERR!   in workspace: admin-ui 
npm ERR!   at location: /Users/aking/workspace/ethyca/fides/clients/admin-ui
```

which is very nice 😎